cli/exchange/business.go

```diff
@@ -0,0 +1,146 @@
+package exchange
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-horizon/anax/businesspolicy"
+	"github.com/open-horizon/anax/cli/cliconfig"
+	"github.com/open-horizon/anax/cli/cliutils"
+	"github.com/open-horizon/anax/exchange"
+	"github.com/open-horizon/anax/externalpolicy"
+	"net/http"
+)
+
+//BusinessListPolicy lists all the policies in the org or only the specified policy if one is given
+func BusinessListPolicy(org string, credToUse string, policy string) {
+	cliutils.SetWhetherUsingApiKey(credToUse)
+	org, credToUse = cliutils.TrimOrg(org, credToUse)
+
+	//get policy list from Horizon Exchange
+	var policyList exchange.GetBusinessPolicyResponse
+	httpCode := cliutils.ExchangeGet(cliutils.GetExchangeUrl(), "orgs/"+org+"/business/policies"+cliutils.AddSlash(policy), cliutils.OrgAndCreds(org, credToUse), []int{200, 404}, &policyList)
+	if httpCode == 404 && policy != "" {
+		cliutils.Fatal(cliutils.NOT_FOUND, "Policy %s not found in org %s", policy, org)
+	}
+
+	jsonBytes, err := json.MarshalIndent(policyList.BusinessPolicy, "", cliutils.JSON_INDENT)
+	if err != nil {
+		cliutils.Fatal(cliutils.JSON_PARSING_ERROR, "failed to marshal 'hzn exchange business listpolicy' output: %v", err)
+	}
+	fmt.Println(string(jsonBytes))
+}
+
+//BusinessAddPolicy will add a new policy or overwrite an existing policy byt he same name in the Horizon Exchange
+func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath string) {
+	cliutils.SetWhetherUsingApiKey(credToUse)
+	org, credToUse = cliutils.TrimOrg(org, credToUse)
+
+	//read in the new business policy from file
+	newBytes := cliconfig.ReadJsonFileWithLocalConfig(jsonFilePath)
+	var policyFile exchange.ExchangeBusinessPolicy
```

Comment:
Please use businesspolicy.BusinessPolicy structure here instead of exchange.ExchangeBusinessPolicy.